src/components/TrackNewExercise.tsx

```diff
@@ -349,6 +353,41 @@ export function TrackNewExercise(props: TrackNewExerciseProps): JSX.Element {
                 required
               />
             </div>
+            <div className="form-group">
+              <label>
+                <strong>Introduction</strong> (optional)
```

Comment:
I've put the introduction field as the last field on the page, as it is the only optional field.

Reply:
OK. We might want to change that later, but fine for now!

Reply:
Sure. Let me know where to put the field.